govulndb/govulndb.go

```diff
@@ -0,0 +1,69 @@
+package govulndb
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"golang.org/x/xerrors"
+
+	pb "gopkg.in/cheggaaa/pb.v1"
+
+	"github.com/aquasecurity/vuln-list-update/utils"
+)
+
+const (
+	baseURL     = "https://storage.googleapis.com/go-vulndb"
+	goVulnDBDir = "go"
+	concurrency = 5
+	wait        = 0
+	retry       = 5
+)
+
+func Update() error {
+	log.Println("Fetching GoVulnDB data...")
+
+	b, err := utils.FetchURL(baseURL+"/index.json", "", retry)
+	if err != nil {
+		return err
+	}
+	var vulnerablePackages map[string]string
+	if err := json.Unmarshal(b, &vulnerablePackages); err != nil {
+		return xerrors.Errorf("failed to decode goVulnDB index.json response: %w", err)
+	}
+	urls := make([]string, len(vulnerablePackages))
+	i := 0
+	for packageName := range vulnerablePackages {
+		url := fmt.Sprintf("%s/%s.json", baseURL, packageName)
+		urls[i] = url
+		i++
+	}
```

Comment:
`vulnerablePackages` is small enough. We can simply use `append` for simplicity .in this case.

```suggestion
	var urls []string
	for packageName := range vulnerablePackages {
		url := fmt.Sprintf("%s/%s.json", baseURL, packageName)
		urls = append(urls, url)
	}
```